.textlintrc

```diff
@@ -27,14 +27,12 @@
         "js",
         "/Loadsmart/i",
         "localhost",
+        "Morocho",
```

Comment:
Unbelievable, I do not believe that this lib does not support Morocho. 😂

Reply:
@davidsonfellipe `--fix` changed it to `Marocco` 🤦‍♂️